### PR TITLE
fix: Only shows verify button when unverified

### PR DESCRIPTION
**Pull request by CarlosLVar**
@migonzalvar

### CODE REVIEW

File: taiga/users/templates/emails/registered_user-body-html.jinja [hunk 1]
```diff
@@ -3,12 +3,18 @@
 {% block body %}
 <table border="0" width="100%" cellpadding="0" cellspacing="0" class="table-body">
     <tr>
-        {% trans signature=sr("signature"), product_name=sr("product_name"), url=resolve_front_url("verify-email", user.email_token), support_email=sr("support.email") %}
+        <td>
+            {% if user.verified_email %}
```
Comment (migonzalvar) [inline taiga/users/templates/emails/registered_user-body-html.jinja:7]: It is the opposite, isn't it?

             {% if not user.verified_email %}

Also we can move the if so it wraps the `<td>`

Comment (CarlosLVar) [reply taiga/users/templates/emails/registered_user-body-html.jinja:7]: Indeed it is! I'll apply those fixes now

File: taiga/users/templates/emails/registered_user-body-text.jinja [hunk 2]
```diff
@@ -1,12 +1,17 @@
-{% trans signature=sr("signature"), product_name=sr("product_name"), url=resolve_front_url("verify-email", user.email_token), support_email=sr("support.email") %}
+{% trans %}
```
Comment (migonzalvar) [inline taiga/users/templates/emails/registered_user-body-text.jinja:1]: product_name undefined here

File: taiga/users/templates/emails/registered_user-body-text.jinja [hunk 3]
```diff
@@ -1,12 +1,17 @@
...
 We're thrilled you've joined our growing community of professionals revolutionizing their way of working and hope you enjoy it.
+{% endtrans %}
 
+{% if not user.verified_email %}
+{% trans url=resolve_front_url("verify-email", user.email_token) %}
 Please confirm your email: {{ url }}
+{% endtrans %}
+{% endif %}
 
+{% trans signature=sr("signature"), product_name=sr("product_name"), support_email=sr("support.email") %}
```
Comment (migonzalvar) [inline taiga/users/templates/emails/registered_user-body-text.jinja:13]: product_name unused

**Pull request merged**